### DISCUSSION TITLE
Welcome to vim-table-mode Discussions!

### DISCUSSION THREAD

**Discussion by dhruvasagar** [dhruvasagar/vim-table-mode#197] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by xz6012104** [comment]
hi,i'm xz from china
I have some problems, Can you help me solve them?
what can i do when i want to input “|” in a cell，or  when i want to wrap in a cell？
best regards

**Reply by dhruvasagar** [reply]
As of now table mode does not support either of these use cases. Wrapping can however be done manually but might be tricky to maintain.

**Comment by LaderBen** [comment]
How to install the vim-table-mode by lazy.nvim, well, I mean I installed the vim-table-mode by lazy.nvim and try to type :TableModelToggle to enable the table mode, however, it shows "Not an editor command: TableModeToggle"

**Reply by dhruvasagar** [reply]
@LaderBen Kindly share your lazy.nvim config for installing vim-table-mode, from the error it implies that table-mode plugin has not been loaded
